sakura_core/window/CMainStatusBar.h

```diff
@@ -56,6 +56,7 @@ class CMainStatusBar : public CDocListenerEx{
 
 	//設定
 	bool SetStatusText(int nIndex, int nOption, const WCHAR* pszText, size_t textLen = SIZE_MAX);
+	void ShowProgressBar(bool bShow) const;
```

Comment:
これも指摘じゃないですが、こんな考え方もあります。
```suggestion
	void ShowProgressBar(bool bShow = true) const;
```
こうしておくと、表示するとき「引数なし」で呼び出せるようになります。
「表示する」がメイン機能で、「引数falseを与えれば非表示にもできる」になります。

メリットは「ShowProgressbar(true);のtrueってなんじゃ～？」を確認する必要がなくなることです。